### DISCUSSION TITLE
"Received backfill packet out of sequence"

### DISCUSSION THREAD

**Discussion by GLoren1** [NightscoutFoundation/xDrip#2207] [Q&A]
Greetings, 

I have been trying to successfully add a calibration to xDrip.  When I try, the calibration does not appear to be accepted. Instead, a message in the logs states "received backfill packet out of sequence", and the readings are not adjusted. 

Perhaps related to this issue, the red "low" indicator color on the xDrip graph is about 30 point above the level that I have set for "low" alerts.   

Any help would be appreciated.

**Comment by Navid200** [comment]
Would you please post a screenshot of the system status page?
You can blackout the transmitter ID if you like.

**Comment by GLoren1** [comment]
Sure. They are attached. 


![temp](https://user-images.githubusercontent.com/62320462/177383633-036e6c1f-f8a7-4f4d-a60c-26acee70db01.jpg)
![temp2](https://user-images.githubusercontent.com/62320462/177383636-1c629f1b-ee2e-4788-b06d-d4697feb4a2f.jpg)

**Reply by Navid200** [reply]
When was the first time you experienced this?  After using your custom xDrip build? 

Did this happen after you attempted to calibrate after a sensor restart?

Can you show the main screen when this happened showing how flat the trend was and how far off your attempted calibration was?

**Reply by GLoren1** [reply]
Sorry for the long delay.  The problem seemed to resolve as I was able to enter new calibration values successfully; however, there is a possibly related problem. That is, when I start new sensors the readings are much higher than expected.  For example, I started a new sensor today and the reading after two hours i252 mg/dl while the reading by meter testing is 149. 

 Also, as noted in my initial post, the red "low" indicator color on the xDrip graph is about 30 point above the level that I have set for "low" alerts

**Reply by Navid200** [reply]
Why are you using a custom xDrip?
Where did you get it from?
Why aren't you use the official xDrip?

**Reply by GLoren1** [reply]
I'm using the version created by @bigdigital because it works with Mi Band.

**Reply by Navid200** [reply]
You don't need to do that any more.  Mi Band is now supported with a companion app:  https://bigdigital.home.blog/2022/06/16/watchdrip-a-new-application-for-xdrip-watch-integration/

This provides all the information you need about the limitations one may face when calibrating a G6:
https://navid200.github.io/xDrip/docs/Calibrate-G6.html

**Reply by GLoren1** [reply]
Thank you.  To revert back to the official xDrip, will I need to uninstall the forked version currently on my phone, or can I upgrade like usual? 

Thanks again

**Reply by Navid200** [reply]
I'm not certain the calibration problem is related to your xDrip.  After you stated what it was, I wanted to point out you don't have to use it.
The only way to switch to the official version is to uninstall first.  But, if you do, you will lose the history unless you carefully go through this:  https://navid200.github.io/xDrip/docs/New-Phone.html

**Reply by GLoren1** [reply]
Thanks. BTW, after about 12 hours after inserting the new sensor, the readings seem to be converging toward the readings from my meter.   In other words, they have been steadily falling.

**Reply by Navid200** [reply]
This is normal behavior after a restarted sensor.
It could also possibly happen, with a brand new sensor, if you enter the 4-digit sensor calibration code incorrectly.
